packages/headless/src/controllers/commerce/product-listing/headless-product-listing.ssr.ts

```diff
@@ -0,0 +1,23 @@
+import {CommerceEngine} from '../../../app/commerce-engine/commerce-engine';
+import {ControllerDefinitionWithoutProps} from '../../../app/ssr-engine/types/common';
+import {ProductListing, buildProductListing} from './headless-product-listing';
+
+export type {ProductListingState as ProductListState} from './headless-product-listing';
+export type ProductList = Pick<ProductListing, 'state' | 'subscribe'>;
```

Comment:
Why do we only pick the `state` and `subscribe` ?

Reply:
I still don't understand why we are renaming `productListing` to `productList`. This will break the ssr-progress script,

Reply:
Thanks for the explanation. I understand better now. So we only want to do this when the controller has subControllers. This only includes the productListing and Search right ?